pom.xml

```diff
@@ -134,40 +221,5 @@
             <version>4.12</version>
```

Comment:
Some of the common dependencies should also be specified in the non-profile part of the `pom.xml` - e.g. junit, graal-sdk, etc. The `<dependencies>` section on GraalVM shall be ideally empty.

Reply:
Not sure what you mean... junit is in the non-profile part of the pom.